iotdb-core/node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -138,6 +139,12 @@ data_replication_factor=1
 # Datatype: Boolean
 # enable_auto_leader_balance_for_iot_consensus=true
 
+# Whether to enable auto leader balance for FastIoTConsensus protocol.
+# The ConfigNode-leader will balance the leader of FastIoTConsensus-RegionGroups by leader_distribution_policy if set true.
+# Notice: Default is true because the FastIoTConsensus depends on this function to distribute leader.
+# Datatype: Boolean
+# enable_auto_leader_balance_for_fast_iot_consensus=true
```

Comment:
It seems that the enable_auto_leader_balance_for_iot_consensus value can be used for both new consensus protocols, so we will not add this configuration